CONTRIBUTING.md

```diff
@@ -436,13 +436,35 @@ and follow below steps to add a rule for azure rest api specs.
 - add a doc for the rule in 'docs', the rule doc file name is following kebab-case, contains following content:
 
 ```md
-# RuleName
+# <add RuleName>
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
```

Comment:
On the same topic as Akhila, this section should be optional because data plane linter rules would not have ARM codes

Reply:
The whole section is not mandate, we just want to standardize the format to be in sync with all other .md files

Reply:
you can mention explicitly that this is mandatory when authoring an ARM guideline

<add RPC code for ARM guidelines, example: "-RPC-Uri-V1-04". This is not required for data plane guidelines>

Reply:
Changed